clusterloader2/api/extensions.go

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
Why is this in this file? Isn't it already merged?

Reply:
It is merged. After rebasing this PR, it looks like it's fixed.
I'm not sure why this happened.